rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/internal/LRARecoveryModule.java

```diff
@@ -260,11 +318,12 @@ private boolean getUids(InputObjectState aa_uids) {
      *
      * @param uids the uids to iterate over
      * @param consumer the consumer that should be called for each Uid
+     * @param transactionType within the Object Store for LRAs.
```

Comment:
A minor, I'm not sure about the `dot` a the end when the sentence does not start with upper case.
Btw. maybe it could be good to add in the comment that the parameter is used only for logging purposes?

Reply:
ok